docs/getting-started/queries/gosdk/complex-data-query.md

```diff
@@ -1,5 +1,1312 @@
 ---
 id: complex-data-query
-title: Query Data Using JSON Fields
+title: Query Data Using Fields in JSON value
```

Comment:
in a JSON Object

Reply:
Done. I never knew till now that Object is the correct word :-)